assets/wizards/advertising/views/suppression/index.js

```diff
@@ -1,80 +1,132 @@
 /**
  * WordPress dependencies
  */
+import apiFetch from '@wordpress/api-fetch';
+import { useState, useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { ToggleControl } from '@wordpress/components';
+import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import {
+	Card,
+	Button,
 	CategoryAutocomplete,
 	SectionHeader,
 	Waiting,
 	withWizardScreen,
 } from '../../../../components/src';
 
-const Suppression = ( { config, onChange } ) => {
+const Suppression = () => {
+	const [ inFlight, setInFlight ] = useState( false );
+	const [ config, setConfig ] = useState( false );
+	const [ postTypes, setPostTypes ] = useState( [] );
+	const fetchConfig = () => {
+		apiFetch( { path: '/newspack-ads/v1/suppression' } ).then( setConfig );
+	};
+	const fetchPostTypes = () => {
+		return apiFetch( {
+			path: addQueryArgs( '/wp/v2/types', { context: 'edit' } ),
+		} ).then( result => {
+			setPostTypes(
+				Object.values( result )
+					.filter( postType => postType.viewable === true && postType.visibility?.show_ui === true )
+					.map( postType => ( {
+						value: postType.slug,
+						label: postType.name,
+					} ) )
+			);
+		} );
```

Comment:
No error handling?

Reply:
Fixed in 46f4755acee4a213a66580ba287cc1c7bc48a3b5